From cab670c726aeb58556b12c703ad9359e5a9b921d Mon Sep 17 00:00:00 2001
From: Jacob Stiffler <j-stiffler@ti.com>
Date: Wed, 30 Nov 2016 13:23:23 -0500
Subject: [PATCH 02/43] ARM: dts: am335x-boneblack-iot-cape: add custom device
 tree

* This device tree enables various HW interfaces such as SPI

Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 arch/arm/boot/dts/am335x-boneblack-iot-cape.dts | 15 +++++++++++++++
 1 file changed, 15 insertions(+)
 create mode 100644 arch/arm/boot/dts/am335x-boneblack-iot-cape.dts

diff --git a/arch/arm/boot/dts/am335x-boneblack-iot-cape.dts b/arch/arm/boot/dts/am335x-boneblack-iot-cape.dts
new file mode 100644
index 0000000..0a5f9d8
--- /dev/null
+++ b/arch/arm/boot/dts/am335x-boneblack-iot-cape.dts
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2016 Texas Instruments Incorporated - http://www.ti.com/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "am335x-boneblack.dts"
+#include "am335x-boneblack-spi0.dtsi"
+
+/ {
+	model = "TI AM335x BeagleBone Black for LPRF";
+	compatible = "ti,am335x-bone-black", "ti,am335x-bone", "ti,am33xx";
+};
-- 
1.9.1

